### ISSUE TITLE
kl项

### ISSUE THREAD

**Issue by NieSYsc20** [hkust-nlp/Laser#4]
Hi. I noticed that both KL_LOSS_COEF=0.001 and KL_COEF=0.001 are set in the training script. Does this mean that the KL term is being applied in both the reward function and the loss function?
https://github.com/hkust-nlp/Laser/blob/c9e7c4ef18192212574c5f6709d67c74baea7605/scripts/training/laser-d-1.5b/laser-d-1.5b-l2048.sh#L36
https://github.com/hkust-nlp/Laser/blob/c9e7c4ef18192212574c5f6709d67c74baea7605/scripts/training/laser-d-1.5b/laser-d-1.5b-l2048.sh#L42

**Comment by VPeterV** [comment]
Hi, thanks for your question!

In VERL, the default setting for GRPO disables the KL term in the reward function. As a result, the kl_coef parameter has no effect under this setting.
